### DISCUSSION TITLE
how many free recognitions in your experience?

### DISCUSSION THREAD

**Discussion by eylenburg** [aleksey-saenko/MusicRecognizer#69] [General]
>There is also the option to use the app without a token, but please note that this will significantly restrict the number of daily recognitions that can be performed. Please keep in mind that this behavior is not guaranteed by the service and can be restricted at any time.

How many free recognitions per day is that approximately? 1, 5, 20?

**Comment by aleksey-saenko** [comment]
It depends on the service.
If we're talking about AudD (default choice mentioned in readme), they don't declare this anywhere. From my own experience, it's about 10 requests per day. Audile v1.5.3 uses up to 3 internet requests in one session (4-8-15 sec).
If we're talking about ACRCloud, they declare a recognition requests limit of 100 for the free plan (per day? per month?). I have never managed to reach their limit so far.
